### PR TITLE
#959 add a max-height and center element with alignItems: center

### PR DESCRIPTION
**Pull request by wcastand**
Issue: #959 

I'm not sure i did the things right (never used lerna so i was a bit lost ). Let me know if i missed something.

## What I did

I added some style to the tabtitles to center his children.

## How to test

In the example, add a react component as a Title with an `button` for example and some padding or margin.

### CODE REVIEW

File: packages/storybook-ui/src/modules/ui/components/down_panel/style.js [hunk 1]
```diff
@@ -26,6 +26,11 @@ export default {
   tabbar: {
     display: 'flex',
     flexWrap: 'wrap',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    maxHeight: 60,
```
Comment (UsulPro) [inline packages/storybook-ui/src/modules/ui/components/down_panel/style.js]: When this at the right side and there are a lot of tabs - they go in several rows. So maxHeight will break it. Better to put it to the tab style individually.

**Comment by UsulPro** [comment]
@wcastand thanks for PR.

Consider applying maxHeight and overflow to the `tablink` style

**Comment by wcastand** [comment]
I put max-Height in the tablink style.
I didn't know how to test it on the right panel in the example of storybook-ui, so let me know if i miss something again.

Thanks for the help on this :)

File: packages/storybook-ui/src/modules/ui/components/down_panel/style.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ export default {
...
+    overflow: 'hidden',
```
Comment (UsulPro) [inline packages/storybook-ui/src/modules/ui/components/down_panel/style.js]: just remove it from here:
```
maxHeight: 60,
overflow: 'hidden',
```

**Review by usulpro — CHANGES_REQUESTED** [review]
remove
```
maxHeight: 60,
overflow: 'hidden',
```
from `tabbar`

**Comment by wcastand** [comment]
I think we're good now :)

**Review by ndelangen — APPROVED** [review]
Looks good to me, tested locally

**Pull request merged**